extensions/wikia/InstantGlobals/InstantGlobalsModule.class.php

```diff
@@ -45,8 +45,16 @@ private function getVariablesValues() {
 		return (object) $ret;
 	}
 
-	public function getScript( ResourceLoaderContext $context ) {
-		$variables = $this->getVariablesValues();
+	public function getScript( ResourceLoaderContext $context = null ) {
+		global $wgMemc;
+		$variables = null;
+		$cacheKey = wfMemcKey( 'instantGlobalsVariablesValues' );
+
+		if ( !( $variables = $wgMemc->get( $cacheKey ) ) ) {
```

Comment:
You can use `WikiaDataAccess:cache` instead: https://github.com/Wikia/app/blob/538336fcb489015585fdde30e1d19a11eca1b127/includes/wikia/WikiaDataAccess.class.php#L52-L52

Reply:
Thanks, changed.

Reply:
Don't assign variables in the if statement please. It's much less readable